src/component/toolbox/feature/DataView.ts

```diff
@@ -352,22 +360,24 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
         }
         else {
             // Use default textarea
-            viewMain.appendChild(textarea);
             textarea.readOnly = model.get('readOnly');
-            textarea.style.cssText = 'width:100%;height:100%;font-family:monospace;font-size:14px;line-height:1.6rem;';
-            textarea.style.color = model.get('textColor');
-            textarea.style.borderColor = model.get('textareaBorderColor');
-            textarea.style.backgroundColor = model.get('textareaColor');
+            const style = textarea.style;
+            // eslint-disable-next-line max-len
+            style.cssText = 'width:100%;height:100%;font-family:monospace;font-size:14px;line-height:1.6rem;resize:none';
+            style.color = model.get('textColor');
+            style.borderColor = model.get('textareaBorderColor');
+            style.backgroundColor = model.get('textareaColor');
             textarea.value = result.value;
+            viewMain.appendChild(textarea);
         }
 
         const blockMetaList = result.meta;
 
         const buttonContainer = document.createElement('div');
-        buttonContainer.style.cssText = 'position:absolute;bottom:0;left:0;right:0;';
+        buttonContainer.style.cssText = 'position:absolute;bottom:5px;left:0;right:0';
```

Comment:
Is `5px` a tiny change on the visual?

Reply:
No. I changed to use [`padding`](https://github.com/apache/echarts/blob/8ad3b74c508c2ff8342e6e3e1aad6337404cfdfa/src/component/toolbox/feature/DataView.ts#L335) instead of [`top:5px;right:5px;bottom:5px;left:5px`](https://github.com/apache/echarts/blob/next/src/component/toolbox/feature/DataView.ts#L327). This `bottom:5px` is to keep the previous style.